central/reports/scheduler/schedule.go

```diff
@@ -278,7 +278,8 @@ func (s *scheduler) sendReportResults(req *ReportRequest) error {
 		return err
 	}
 	// Format results into CSV
-	zippedCSVResult, err := common.Format(reportData, nil, "")
+	includeNvd := rc.GetVulnReportFilters().GetIncludeNvdCvss()
+	zippedCSVResult, err := common.Format(reportData, nil, "", includeNvd)
```

Comment:
You can just send `includeNvd` as `false` here. This is v1 report implementation. I will create a ticket to remove this v1 reporting code.